src/wwwroot/controllers/TemplateCtrl.js

```diff
@@ -25,22 +26,60 @@
 
     $scope.loadInProgress = false;
     $scope.saveInProgress = false;
+    $scope.isLegacyTemplate = true;
 
-    $scope.saveHtmlRaw = saveHtmlRaw;
+    $scope.save = save;
     $scope.saveAndEditWithMseditor = saveAndEditWithMseditor;
 
     if ($routeParams["templateId"]) {
       load($routeParams["templateId"]);
     }
+    else {
+      $scope.isLegacyTemplate = !$rootScope.forceMsEditor;
+    }
 
     /*************************/
     /* Function declarations */
     /*************************/
 
     function load(templateId) {
       $scope.loadInProgress = true;
-
-      return templates.getTemplateWithBody(templateId).then(function (result) {
+      return templates.getTemplate(templateId).then(function (template) {
+        switch (template.bodyType) {
+          case "empty":
+            $scope.isLegacyTemplate = false;
+            break;
+          case "mseditor":
+            $scope.isLegacyTemplate = false;
+            break;
+          case "rawHtml":
+            $scope.isLegacyTemplate = true;
+            break;
+        }
+        if($rootScope.forceMsEditor && !$scope.isLegacyTemplate)
```

Comment:
Why do you need to test _forceMsEditor_ value here?

Reply:
Sorry this is not neccessary, I will remove it.